test/cql-pytest/test_tools.py

```diff
@@ -948,3 +949,16 @@ def test_scylla_sstable_shard_of(cql, test_keyspace, scylla_path, scylla_data_di
             expected_json = [shard_id]
             for actual_json in sstables_json.values():
                 assert actual_json == expected_json
+
+
+def test_scylla_sstable_no_args(scylla_path):
+    res = subprocess.run([scylla_path, "sstable"], capture_output=True, text=True)
+
+    sys.stdout.write(res.stdout)
+    sys.stderr.write(res.stderr)
```

Comment:
These lines look like something you added during debugging and forgot to remove. The assert code below will anyway print the stdout or stderr if one of them doesn't match the expectation.

Reply:
Good point, I will remove these.

Reply:
Removed in v2.